views/lexicon/tabs/modals.html.twig

```diff
@@ -40,7 +40,7 @@
 
 <p><a data-toggle="modal" href="#myModal" class="btn">Launch Standard Modal</a></p>
 
-<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
Did `aria-labelledby` throw an info/warning?

Reply:
There is no element with id="myModalLabel". Let me know if you wish to bind this to a different element. (Ref: https://www.w3.org/TR/WCAG20-TECHS/ARIA7.html Example 2)

Reply:
It's probably the modal trigger which needs the ID?

Reply:
Just checked it doesn't affect any function of the modal